pkg/scheduler/scheduler.go

```diff
@@ -399,6 +399,9 @@ func (sched *Scheduler) bindVolumes(assumed *v1.Pod) error {
 		if forgetErr := sched.config.SchedulerCache.ForgetPod(assumed); forgetErr != nil {
 			klog.Errorf("scheduler cache ForgetPod failed: %v", forgetErr)
 		}
+		// Volumes may be bound by PV controller asynchronously, we must clear
+		// stale pod binding cache.
+		sched.config.VolumeBinder.DeletePodBindings(assumed)
```

Comment:
is there a test we can set up that would exercise the bug this fixes?

Reply:
hard to set up a test case to cover this, probably we can add a stress test, it's easier to reproduce this issue in a few runs locally

Reply:
+1 to async "stress" test... this looks like it should be fairly quick to trigger

Reply:
We do have some existing stress integration tests but I think they don't handle prepound PVs.  It shouldn't be too difficult to randomly make a fraction of the PVs prebound